lib/Map/UserDrawing.js

```diff
@@ -0,0 +1,400 @@
+'use strict';
+
+import MapInteractionMode from '../Models/MapInteractionMode';
+
+var DeveloperError = require('terriajs-cesium/Source/Core/DeveloperError');
```

Comment:
Probably good to avoid mixing styles like this in a single flie; either use `import` or `require` throughout.